components/feature_restricted_modal/feature_restricted_modal.tsx

```diff
@@ -20,64 +20,65 @@ import {isModalOpen} from 'selectors/views/modals';
 import {GlobalState} from 'types/store';
 import {ModalIdentifiers, AboutLinks, LicenseLinks} from 'utils/constants';
 
-import './create_team_restricted_modal.scss';
+import './feature_restricted_modal.scss';
 
-const CreateTeamRestrictedModal = () => {
+type FeatureRestrictedModalProps = {
+    modalTitle: string;
+    modalMessage: string;
+    modalTitleAfterTrial: string;
+    modalMessageAfterTrial: string;
+}
+
+const FeatureRestrictedModal = ({
```

Comment:
Changed this to be generic and used in advance for restricted features.

Reply:
Any tests we can add to this generic modal now?